wasm/main.py

```diff
@@ -2376,8 +2365,8 @@ def spec_allocfunc(S: Store,
     logger.debug('spec_allocfunc()')
 
     funcaddr = FunctionAddress(len(S.funcs))
-    func_type = module.types[func.type_idx]
-    funcinst = FunctionInstance(func_type, module, func)
+    function_type = module.types[func.type_idx]
+    funcinst = FunctionInstance(function_type, module, func)
```

Comment:
Looks like all the other `func` variables inside this method could be updated as well.

Reply:
To avoid the merge conflict I'm going to leave as is since this gets cleaned up in https://github.com/ethereum/py-wasm/pull/59/files#diff-26f7706e7df969dc490d122a6f78ae05R55